roles/cfg_openwrt/templates/corerouter/config/lldpd.j2

```diff
@@ -0,0 +1,13 @@
+#jinja2: trim_blocks: "true", lstrip_blocks: "true"
+config lldpd config
+	option enable_cdp 0
+	option enable_fdp 0
+	option enable_sonmp 0
+	option enable_edp 0
+	option lldp_class 4
+	option lldp_location "2:FR:6:Commercial Rd:3:Roseville:19:4"
```

Comment:
This should be properly filled with the location of the device.

Reply:
I think we can just delete this line.